src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/handlers/common/CorrelationKeys.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.domain.handlers.common;
+
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode
+public class CorrelationKeys {
```

Comment:
is this a marker class  ?

Reply:
Yes, it's is. I use things like sendMessage<? extend CorrleationKeys>  it's a way to control what we can pass in there.